lib/components/base-components/NormalComponent.ts

```diff
@@ -189,7 +189,7 @@ export class NormalComponent<
     const { schematicSymbolName } = this.config
     if (!schematicSymbolName) return
     // TODO switch between horizontal and vertical based on schRotation
-    const symbol_name = `${this.config.schematicSymbolName}_horz`
+    const symbol_name = `${this.config.schematicSymbolName}`
```

Comment:
Hmm i think you need this

Reply:
this is why you're getting the type error, symbols always have _horz or _vert as part of their name

Reply:
this is probably fine